database/dynamodb.go

```diff
@@ -29,34 +21,50 @@ type AWSCredentials struct {
 var db *dynamodb.DynamoDB
 
 func init() {
+	defer func() {
+		if err := recover(); err != nil {
+			fmt.Println(err)
+		}
+	}()
 
-	env, found := os.LookupEnv(Constants["ENV"])
-	if !found {
-		// load the env values using the `setUpEnv` function.
-		utils.SetUpEnv()
+	env := os.Getenv(utils.ENV)
+	if env == utils.PROD {
+		log.Println(env, " is the env")
 	}
 
 }
 
 func getAWSCredentials() *AWSCredentials {
```

Comment:
this function does not check if the environment variables utils.REGION, utils.ACCESS_KEY, and utils.SECRET_KEY are set before calling os.LookupEnv(). If these environment variables are not set, then os.LookupEnv() will return an empty string. This will cause the function to panic.

can you add below checks
if os.Getenv(utils.REGION) == "" {
	log.Panic("AWS Region not set")
}
if os.Getenv(utils.ACCESS_KEY) == "" {
	log.Panic("AWS access key not set")
}
if os.Getenv(utils.SECRET_KEY) == "" {
	log.Panic("AWS secret key not set")
}

Reply:
rather than getenv directly use os.LookupEnv, it returns variable and a bool stating if the variable is found or not
```go
envVar, found := os.LookupEnv("ritik")
```

Reply:
![image](https://github.com/Real-Dev-Squad/feature-flag-backend/assets/57758447/90dbc0c6-d3fb-4a88-b2ac-a91d30dfdd5e)

Reply:
Added the checks and updated the code to use `LookupEnv`